Git Formats/Git Log.sublime-syntax

```diff
@@ -39,6 +39,12 @@ contexts:
               scope: entity.other.branch-name.git.log
             - match: ','
               scope: punctuation.separator.sequence.git.log
+    - match: (?=^diff\s--git)
+      embed: scope:source.shell.bash
+      escape: $
+    - match: (?=^index\s)
+      embed: scope:source.diff
+      escape: (?=^diff\s--git)
```

Comment:
We probably want to escape if we match the next `commit ...` line`?

Reply:
Maybe we should also be abit lazier with regards to whitespace?

```suggestion
      escape: (?=^(commit\s+\h{7,}|diff\s+--git\s))
```

Reply:
good idea, done :)

Reply:
`\h`?

Reply:
Would probably enough.

Reply:
https://github.com/kkos/oniguruma/blob/master/doc/RE#L79